catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -230,7 +230,7 @@
 
   private String srsName;
 
-  private boolean allowRedirects;
+  private boolean allowRedirects = Boolean.FALSE;
```

Comment:
✏️ If we are expecting a `Boolean` to be here instead of a `boolean`, it might be worth changing the variable type.